src/lib/openapi/spec/constraint-schema.ts

```diff
@@ -1,36 +1,57 @@
 import { FromSchema } from 'json-schema-to-ts';
 import { ALL_OPERATORS } from '../../util/constants';
 
-export const constraintSchema = {
-    $id: '#/components/schemas/constraintSchema',
+export const constraintSchemaBase = {
```

Comment:
I've extracted this into a separate object so that we can reuse the base across multiple constraint schemas and keep them in sync. In this case, I want the playground constraint schema to be the exact same as the regular constraint schema, but with some added information, but when you add `additionalProperties: false` on a schema, you can't merge it with others for inheritance. This is a way around that.